dom-manip.js

```diff
@@ -28,10 +37,12 @@ const flightCategories = [
 	"Line Obstacle",
 ];
 
+localStorage.setItem("flights", JSON.stringify([]));
```

Comment:
Separation of concerns. Please separate.

Reply:
Thank you. Issue has been resolved. Please confirm